app/konnect/servicehub/service-documentation.md

```diff
@@ -9,90 +9,27 @@ documentation to the Dev Portal. Publishing services to the Dev Portal is the on
 
 ## Service descriptions
 
-You can provide extended descriptions of your services with a Markdown (`.md`) file. The contents of this file will be displayed as the introduction to your API in the Dev Portal.
+You can provide extended descriptions of your services with a Markdown (`.md`) file. The contents of this file will be displayed as the introduction to your API in the Dev Portal. Service descriptions can be any markdown document that describes your service: 
 
-### Upload a service description
+* Release notes
+* Support and SLA 
+* Business context and use cases
+* Deployment workflows
 
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, then follow these steps:
+<p align="center">
+  <img src="/assets/images/docs/konnect/konnect_service_docs_description.png" />
+</p>
 
-1. In the **Service Document** section, click **Upload Document**.
-
-1. Select a `.md` file to upload.
-
-### Update a service description
-
-You can upload a new document to replace an existing service document.
-
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, then follow these steps:
-
-1. In the **Service Document** section, click the {% konnect_icon markdown %} **file icon**, then **Replace**.
-
-1. Select a new `.md` file to upload.
-
-### Delete a service description
-
-You can delete an existing document from the Dev Portal. Deleting a service description permanently removes it from the Dev Portal and the Service Hub.
-
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service, then follow these steps:
-
-1. In the **Service Document** section, click the {% konnect_icon cogwheel %} icon, then **Delete**.
-
-1. Confirm to permanently delete the service description file.
+All service descriptions are managed from the **Documentation** section in the service overview. Once you've uploaded the markdown file, you have a preview of how it will render, the option to edit, and a view of the publication status. You can also create a hierarchy between the docs you upload that will be reflected in the way they're displayed in the Dev Portal.
 
```

Comment:
I think it would be useful for customers to see this visually, so could we add screenshots depicting multiple documents, their hierarchy, and the ability to move them around and switch up the hierarchy?

Reply:
@mptap, thanks for the suggestion, but we're only using screenshots in a limited capacity in the docs, not meant to replace seeing all functionality live in the product.